src/pathwayvis/components/map/map.component.ts

```diff
@@ -496,6 +516,19 @@ class MapComponentCtrl {
     this.$scope.$apply(() =>
       this.processActionClick(this.contextActions[1], this.contextElement));
   }
+
+  public handleChangeBounds(args) {
+    const bounds = {lower: 0, upper: 0};
+    const tooltipContainer = d3.select('div#tooltip-container');
+    bounds.lower = parseInt(tooltipContainer.select('#lowerbound').property("value"));
```

Comment:
It'd be better to pass these down as parameters to the tooltip. Did you look into that?